test/integration/popular_posts_test.rb

```diff
@@ -42,7 +42,7 @@ class PopularPostTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "Unpablished article are not displayed" do
+  test "Unpublished article are not displayed" do
```

Comment:
Which is correct, "unpublished articles are not displayed" or "an unpublished artcile is not displayed"?